Octokit.Reactive/Clients/ObservableRepositoryContentsClient.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Reactive.Threading.Tasks;
+using Microsoft.SqlServer.Server;
```

Comment:
Uhm.......😲

I have no idea where that would have come from...I will of course remove said reference quicker than quick 😁
________________________________
From: Brendan Forster<mailto:notifications@github.com>
Sent: ‎23/‎04/‎2015 3:10
To: octokit/octokit.net<mailto:octokit.net@noreply.github.com>
Cc: Henrik Andersson<mailto:henrik.a.andersson@outlook.com>
Subject: Re: [octokit.net] Repository Get Archive Link (#765)

> @@ -1,5 +1,6 @@
>  ﻿using System;
>  using System.Reactive.Threading.Tasks;
> +using Microsoft.SqlServer.Server;

Wait, what?

---
Reply to this email directly or view it on GitHub:
https://github.com/octokit/octokit.net/pull/765/files#r28893592